src/internal.c

```diff
@@ -10875,6 +10884,13 @@ int ProcessPeerCerts(WOLFSSL* ssl, byte* input, word32* inOutIdx,
                 ret = ssl->error = 0;
             }
 
+            if (ret != 0) {
+                if (!ssl->options.verifyNone) {
+                    DoCertFatalAlert(ssl, ret);
```

Comment:
@toddouska: The original code did not exit after sending the fatal alert, so I kept the existing behavior here. Current code keeps processing certs and fails later after the peer cert is processed. Thinking about this and wondering why we wouldn't just exit the ParseCertRelative function with the error after sending a fatal alert, since the TLS is shutting down/irrecoverable?